jaxb-ri/codemodel/pom.xml

```diff
@@ -28,5 +30,59 @@
         <module>codemodel</module>
         <module>codemodel-annotation-compiler</module>
     </modules>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.0</version>
```

Comment:
I've stumbled over https://github.com/apache/maven-compiler-plugin/blob/master/src/it/multirelease-patterns/singleproject-runtime/pom.xml#L98 - maybe you could adopt it (w/o the mr related part) ?

Reply:
Yes it would be good to replace current pattern in all projects with it.